plugins/out_opentelemetry/opentelemetry.c

```diff
@@ -689,29 +693,38 @@ static int process_logs(struct flb_event_chunk *event_chunk,
                         struct flb_input_instance *ins, void *out_context,
                         struct flb_config *config)
 {
-    Opentelemetry__Proto__Logs__V1__LogRecord *log_record_list[FLB_LOG_RECORD_BATCH_SIZE];
-    Opentelemetry__Proto__Logs__V1__LogRecord log_records[FLB_LOG_RECORD_BATCH_SIZE];
-    Opentelemetry__Proto__Common__V1__AnyValue log_bodies[FLB_LOG_RECORD_BATCH_SIZE];
+    struct opentelemetry_context *ctx;
+    ctx = out_context;
+
+    // These were initially variable length arrays.
```

Comment:
use /* for comments */, what's the issue number you are referring to here ?

Reply:
#6512 
When it was a variable length array stored on the stack, the event chunk was being corrupted somewhere between 720 & 730 [initialisation](https://github.com/fluent/fluent-bit/blob/46bb7845817a37fef3fc01b9ebd54797c192419b/plugins/out_opentelemetry/opentelemetry.c#L721)